Undefined name 'self' in madry_cifar10_model.py

flake8 testing of https://github.com/tensorflow/cleverhans on Python 2.7.14

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./examples/madry_lab_challenges/cifar10/madry_cifar10_model.py:267:31: F821 undefined name 'self'
    return tf.nn.xw_plus_b(x, self.w, self.b)
                              ^
./examples/madry_lab_challenges/cifar10/madry_cifar10_model.py:267:39: F821 undefined name 'self'
    return tf.nn.xw_plus_b(x, self.w, self.b)
                                      ^
2       F821 undefined name 'self'
2
```

--- a/examples/madry_lab_challenges/cifar10/madry_cifar10_model.py
+++ b/examples/madry_lab_challenges/cifar10/madry_cifar10_model.py
@@ -251,6 +251,7 @@ def _relu(x, leakiness=0.0):
     """Relu, with optional leaky support."""
     return tf.where(tf.less(x, 0.0), leakiness * x, x, name='leaky_relu')
 
+
 def _global_avg_pool(x):
     assert x.get_shape().ndims == 4
     return tf.reduce_mean(x, [1, 2])

--- a/examples/madry_lab_challenges/cifar10/madry_cifar10_model.py
+++ b/examples/madry_lab_challenges/cifar10/madry_cifar10_model.py
@@ -251,22 +251,6 @@ def _relu(x, leakiness=0.0):
     """Relu, with optional leaky support."""
     return tf.where(tf.less(x, 0.0), leakiness * x, x, name='leaky_relu')
 
-
-def _fully_connected(x, out_dim):
-    """FullyConnected layer for final output."""
-    num_non_batch_dimensions = len(x.shape)
-    prod_non_batch_dimensions = 1
-    for ii in range(num_non_batch_dimensions - 1):
-        prod_non_batch_dimensions *= int(x.shape[ii + 1])
-    x = tf.reshape(x, [tf.shape(x)[0], -1])
-    w = tf.get_variable(
-        'DW', [prod_non_batch_dimensions, out_dim],
-        initializer=tf.initializers.variance_scaling(distribution='uniform'))
-    b = tf.get_variable('biases', [out_dim],
-                        initializer=tf.initializers.constant())
-    return tf.nn.xw_plus_b(x, self.w, self.b)
-
-
 def _global_avg_pool(x):
     assert x.get_shape().ndims == 4
     return tf.reduce_mean(x, [1, 2])

